pom.xml

```diff
@@ -495,15 +498,13 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <version>3.4.0</version>
-        <configuration>
-          <additionalparam>-Xdoclint:none</additionalparam>
-        </configuration>
         <executions>
           <execution>
-            <id>attach-javadocs</id>
-            <configuration>
-              <additionalparam>-Xdoclint:none</additionalparam>
-            </configuration>
+            <phase>compile</phase>
+            <id>generate-javadocs</id>
```

Comment:
This now IIUC means we generate javadocs twice during the build pipeline,
Once with the parent invocation bound to package and now here?

This also means we create the javadoc on each ATH split.  In the grand scheme of the ATH times this is probably lost but would we just be better off with a `mvn verify -DskipTests` in a branch on the pipeline?

Reply:
see also https://github.com/jenkinsci/acceptance-test-harness/pull/893

Reply:
`mvn verify -DskipTests` doesn't run javadoc currently by the looks of it

Reply:
https://github.com/jenkinsci/acceptance-test-harness/pull/893 works now